x-pack/plugins/ml/common/types/security.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const REQUIRED_ROLES = ['machine_learning_admin', 'machine_learning_user'];
+export const REQUIRED_LICENSES = ['standard', 'gold', 'trial', 'platinum'];
+export const LICENSES = ['oss', 'basic', 'standard', 'gold', 'trial', 'platinum'];
+export type LicenseType = 'oss' | 'basic' | 'trial' | 'standard' | 'basic' | 'gold' | 'platinum';
```

Comment:
Is there a way of defining these license type values just once, for example in an enum, rather than using the Strings in multiple definitions?